### ISSUE TITLE
Replace "elmentor-" prefix and long class names in HTML and CSS to save 20KB on every page.

### ISSUE THREAD

**Issue by vovkasolovev** [pojome/elementor#6062]
Hi! 
I did a simple text replacement test and found out that you can save about 15-25 KB on a single regular size page if you **just replace** both in HTML and in the corresponding CSS each _"elmentor-"_ with a shorter one like _"e-"_. 

Generated code is overflowing with this prefix and other, like _"-element"_, _"-field-group"_ and so on. I suggest there can be some optimization for HTML and CSS class names. Because, workflow do not assume that we will work with generated class names, I'd prefer if code will be just minimized as it possible.
I'm not talking about duplicated containers, or how to hide the prefix (#4677), I am talking only about code size that can be giant on sites with lot of content. 

Optimization rules can be designed to keep human readable classes names, just make them shorter. I understand, that this can be in touch with some deep hard coding, but guess there will be little problem with supporting backward compatibility of old sites because this can be solved with some synonym system, with will help move stress from every page load to the cheap one time code run.

**Comment by orhancolak** [comment]
This is important to me. If that's done, I can switch to Elementor.

**Comment by pwnoobz** [comment]
I was searching for this feature too. I think I can create a post-processor type of plugin to overwrite them with e- but it will be useless on a bigger site where more requests are made because this will kill the whole purpose of loading speed optimisation.

**Comment by MB-Jan** [comment]
e- would be so nice. I just checked on a random page of my website and it would reduce the source code size from 255KB to 223KB. This does not yet include the savings that would be possible in Elementor's CSS and JS files if the prefix would be changed. It would be great if the Elementor team could consider this feature.

**Comment by mbaierl** [comment]
This request does not make any sense.

Every modern webserver is compressing the data before it is transmitted to the client. So using „elementor-“ many many times on the same page is not going to have any difference over „e-“, since it is compressed and only stored once any way.

The finally difference will be something like 8 bytes … 

The uncompressed version you refer to makes no difference on the wire, but using „e-“ only will cause more conflicts with other plugins and themes.

**Comment by vovkasolovev** [comment]
@mbaierl, I agree, if compression is enabled, the impact may be negligible. But it's not mandatory, and there are plenty of WP sites. So don't just count on it if it can be corrected. 

Also, It's not so much about technical optimization as it is about human optimization. For example, when you create a widget, when you have to analyze the output, and it's hard to read lines that are crowded with the word _elementor-_. 

And if you look broadly, Elementor's competitors have been known to label it "bloated". On the technical side, we understand that this is not a problem, the browser can handle long class names or deep container constructs. But, from a promotional point of view, it would be beneficial to break this label with more and more steps in this direction.
